ts/session/snode_api/swarmPolling.ts

```diff
@@ -119,11 +119,11 @@ export class SwarmPolling {
   ): Promise<Array<any>> {
     const edkey = node.pubkey_ed25519;
 
-    const pkStr = pubkey.key ? pubkey.key : pubkey;
+    const pkStr = (pubkey.key ? pubkey.key : pubkey) as string;
```

Comment:
```suggestion
    const pkStr = PubKey.cast(pubkey).key;
```
and above you can change `pubkey` type to `PubKey | string`

Reply:
But I don't want to change the type... it is just that occasionally the type could not be checked at compile time, and I don't get what I expected :(

Reply:
I think we should still cover that case in the type, that's why we call `PubKey.cast`, if it's a string it'll be converted to a `PubKey` otherwise it'll return the passed in `PubKey`. I've been doing this around code where I know there'll be a lot of js and ts handovers.

Reply:
I changed to `PubKey | string` in the public interface